### DISCUSSION TITLE
Clarity on Migration instructions

### DISCUSSION THREAD

**Discussion by Dvalin21** [InvoiceShelf/InvoiceShelf#192] [Q&A] [answered]
Could you be a little more clear for the migration instructions? Is it just for the manual install version or is it also for docker install version? Thank in advance.

**Comment by rihards-simanovics** [comment, marked as answer]
> Could you be a little more clear for the migration instructions? Is it just for the manual install version or is it also for docker install version? Thank in advance.

Hey @Dvalin21 sorry for this one. So, the migration process has been specifically designed for a traditional hosting, i.e. LAMP/LEMP. For docker it should be simple if it's the docker-compose version you likely be better served by viewing the docker-compose templates under the docker repo but here is an example docker-compose for the [mysql version](https://github.com/InvoiceShelf/docker/blob/master/docker-compose.mysql.yml)

**Reply by rihards-simanovics** [reply]
I am going to work on the docs a little more for the migration process as it seems to confuse quite a few people.

**Comment by theapiarist** [comment]
Is it possible to migrate data from one machine to another, for example by doing a database dump and then re-importing, or is it better to make an image of the entire container and move that? I've found information on backups, and the ability to export/save them, but nothing on how they can be re-imported.

**Reply by rihards-simanovics** [reply]
are you on Docker or traditional LAMP?

**Reply by theapiarist** [reply]
Docker.

**Reply by rihards-simanovics** [reply]
If your old instance was Docker-based, then it's as simple as copying the contents of the volumes and moving them to a new volume.
